test/server.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const Fastify = require('..')
+
+test('listen should accept null port', t => {
+  t.plan(1)
+
+  const fastify = Fastify()
+  t.teardown(fastify.close.bind(fastify))
+  fastify.listen({ port: null }, (err) => {
+    t.error(err)
+  })
+})
+
+test('listen should accept undefined port', t => {
```

Comment:
see above 😄

Reply:
Replied on previous comment. Thanks for the feedback!